tensorflow/compiler/tf2tensorrt/convert/trt_optimization_pass.cc

```diff
@@ -156,54 +58,57 @@ StatusOr<bool> ShouldConvertFunction(const grappler::GrapplerItem& item) {
   if (item.id == "tf_graph") {
     return false;
   }
-  const grappler::GrapplerFunctionItem& func_item =
+  const auto& func_item =
       tensorflow::down_cast<const grappler::GrapplerFunctionItem&>(item);
-  const tensorflow::AttrSlice& attr = func_item.func_attr();
+  auto attr = func_item.func_attr();
```

Comment:
Style guide discourages the use of auto unless the data type is obvious from the RHS. 
line 61 is good, but this line is not. Can you reverse this change?
Please also remember to squash the commits.

Reply:
Fixed, thanks